README.md

```diff
@@ -1,11 +1,6 @@
 
-| Travis |
-| ------ |
-| [![Build Status](https://v3.travis.ibm.com/mqoc/mqcloud-go-sdk.svg?token=xb8AgTFykN966eqXcU5v)](https://v3.travis.ibm.com/mqoc/mqcloud-go-sdk) |
-
-
-# IBM Cloud MQoC Services Go SDK 0.0.1
-Go client library to interact with the various [IBM Cloud MQoC APIs](https://cloud.ibm.com/apidocs).
+# IBM Cloud MQ on Cloud Services Go SDK 0.0.1
+Go client library to interact with the various [IBM Cloud MQ on Cloud APIs](https://cloud.ibm.com/apidocs).
```

Comment:
We wanna give our api doc reference which is 

https://cloud.ibm.com/apidocs/mq-on-cloud

but its not available rightnow. Need to check with the team.